coreapps-web-2.2/src/main/java/org/openmrs/module/coreapps/diagnosis/DiagnosisServiceCompatibility2_2.java

```diff
@@ -0,0 +1,36 @@
+package org.openmrs.module.coreapps.diagnosis;
+
+import java.util.ArrayList;
+
+
+import java.util.List;
+
+import org.openmrs.CodedOrFreeText;
+import org.openmrs.ConditionVerificationStatus;
+import org.openmrs.Encounter;
+import org.openmrs.annotation.OpenmrsProfile;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.emrapi.diagnosis.CodedOrFreeTextAnswer;
+import org.openmrs.module.emrapi.diagnosis.Diagnosis;
+
+
+
+@OpenmrsProfile(openmrsPlatformVersion = "2.2")
+public class DiagnosisServiceCompatibility2_2 implements DiagnosisServiceCompatibility {
+
+	@Override
+	public List<Diagnosis> getPrimaryDiagnosis(Encounter encounter) {
```

Comment:
@ibacher for this pr,let me first write tests for the sub module so as i can easly yell the source of the null value